windows-apps-src/design/basics/navigation-basics.md

```diff
@@ -172,20 +172,31 @@ With few exceptions, any app that has multiple pages uses a frame. Typically, an
         ![tabs and pivot image](images/nav/thumbnail-tabs-pivot.svg)
     :::column-end:::
 	:::column span="2":::
-        [**Top navigation and tabs**](../controls-and-patterns/navigationview.md)
+        [**Top navigation**](../controls-and-patterns/navigationview.md)
 
-Displays a horizontal list of links to pages at the same level. The [NavigationView](../controls-and-patterns/navigationview.md) control implements the top navigation and tabs patterns.
+Displays a horizontal list of links to pages at the same level. The [NavigationView](../controls-and-patterns/navigationview.md) control implements the top navigation pattern.
         
 Use top navigation when:
 
 - You want to show all navigation options on the screen.
 - You desire more space for your app's content.
 - Icons cannot clearly describe your navigation categories.
-		
+
+:::row-end:::
+
+:::row:::
+    :::column:::
+        ![tabs and pivot image](images/nav/thumbnail-tabs-pivot.svg)
+    :::column-end:::
+	:::column span="2":::
+        [**Tabs**](../controls-and-patterns/tab-view.md)
+
+Displays a horizontal set of tabs and their respective content. The [TabView](../controls-and-patterns/tab-view.md) control is useful for displaying several pages (or documents) while giving the user the capability to rearrange, open, or close tabs.
+	
 Use tabs when:
 
-- You want to preserve navigation history and page state.
-- You expect users to switch between tabs frequently.
+- You want users to be able to dynamically open, close, or rearrange pages.
+- You expect users to be able to drag and drop between windows.
```

Comment:
This statement doesn't make sense to me. Isn't that what windows does?

Reply:
I'm open to suggestions on wording. Basically I was trying to say "TabView allows users to drag and drop tabs/pages between windows" which is a differentiator compared to NavigationView or Pivot.

Reply:
What about "You expect users to be able to drag and drop **tabs or pages** between windows." ?

Reply:
Since this is about tab navigation, do we just use 'tab' as the noun vs. page?

You could also add a bullet for 'You expect a large number of tabs open at once.'

For the drag-and-drop (would be cool to link to the sample from here too), maybe 'You expect users to be able to easily move a tab between windows in your application that use tabs.'? (Can we call out 'similar to Microsoft Edge')?